supported_cli_versions.json

```diff
@@ -0,0 +1,8 @@
+[
+  "v2.7.6",
```

Comment:
Nice!

Can you make sure that the most recent, non-nightly version is first in the list? This will make it easier if we want to grab the latest version in other parts of the code. Just need to grab the first element of the array.